akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1802,15 +1804,15 @@ trait FlowOps[+Out, +Mat] {
    * @param maxSubstreams configures the maximum number of substreams (keys)
    *        that are supported; if more distinct keys are encountered then the stream fails
    */
-  def groupBy[K](maxSubstreams: Int, f: Out ⇒ K): SubFlow[Out, Mat, Repr, Closed] = {
+  def groupBy[K](maxSubstreams: Int, f: Out ⇒ K, allowClosedSubstreamRecreation: Boolean = false): SubFlow[Out, Mat, Repr, Closed] = {
```

Comment:
also as the override please; default methods are hard to evolve in a bin compat way if we ever have to add another overload

Reply:
You mean: as the overload? As in: don't use default params but provide an overloaded method?

Reply:
Makes sense, I've been bitten by those often enough. I agree it can become hard to evolve, especially if you start to need overloads with a differing parameter set. Will change!